### PR TITLE
Rename KIA_NIRO_HEV to KIA_NIRO_PHEV

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/sunnyhaibin/openpilot-1/blob/bd4c0fd8357214d4114f4a2151f7c909d3295fcd/selfdrive/car/tests/routes.py#L106)
- [x] route with openpilot: `5975c51414e889de|2022-04-16--12-11-52`
- [x] harness type: [Hyundai C](https://github.com/commaai/openpilot/wiki/Hyundai-Kia-Genesis#harness-guide)

Thanks to community Niro Plug-In Hybrid owner securelyfitz.

### CODE REVIEW

**Comment by sunnyhaibin** [comment]
Hey @sshane - I can't seem to generate CARS.md with the new port. This is what I'm seeing when I run docs.py:
```
Traceback (most recent call last):
  File "selfdrive/car/docs.py", line 73, in <module>
    f.write(generate_cars_md(get_tier_car_info(), args.template))
  File "selfdrive/car/docs.py", line 46, in get_tier_car_info
    _car_info.init(CP, non_tested_cars, ALL_FOOTNOTES)
  File "/data/pythonpath/selfdrive/car/docs_definitions.py", line 59, in init
    assert CP.minSteerSpeed == 0, f"Minimum steer speed set in both CarInfo and CarParams for {CP.carFingerprint}"
AssertionError: Minimum steer speed set in both CarInfo and CarParams for KIA NIRO PLUG-IN HYBRID 2018
```
Is there something I'm missing? Thanks!

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ class CAR:
   KIA_FORTE = "KIA FORTE E 2018 & GT 2021"
   KIA_K5_2021 = "KIA K5 2021"
   KIA_NIRO_EV = "KIA NIRO EV 2020"
-  KIA_NIRO_HEV = "KIA NIRO HYBRID 2019"
+  KIA_NIRO_PHEV = "KIA NIRO HYBRID 2019"
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:62]: From this PR, this car is actually a plug-in hybrid: https://github.com/commaai/openpilot/pull/21481

**Comment by sshane** [comment]
- [ ] Awaiting new test route under the new platform

**Comment by sshane** [comment]
@sunnyhaibin do we have a new route?

**Comment by sunnyhaibin** [comment]
> @sunnyhaibin do you know if we can get a new route?

Will ping the user again to get a route going 👍

**Comment by sshane** [comment]
haha we simplified this full car port PR so much it's not a car port anymore

**Pull request merged**